stories/charts/gv-chart-gauge.stories.js

```diff
@@ -30,6 +30,12 @@ export default {
 const conf = {
   component: 'gv-chart-gauge',
   events,
+  css: `
+  :host {
+    height: 300px;
+    background: var(--gv-theme-neutral-color-dark, #D9D9D9);
```

Comment:
Non blocking: i think it is a bit weird to have a background only for this component. it is good in the PR to demonstrate the fix but as a classical user it may not seem consistency.

Maybe we can add a knob to change the background ?